CITATION.cff

```diff
@@ -0,0 +1,75 @@
+cff-version: 1.2.0
+title: 'SpiceyPy: a Pythonic Wrapper for the SPICE Toolkit'
+message: If you are publishing work that uses SpiceyPy, please cite SpiceyPy and the SPICE toolkit.
```

Comment:
should the message be in quotes? also update to say "cite SpiceyPy using the JOSS doi..." or something to the effect of if citing the use of spiceypy, use the joss doi, but if needing to cite a specific version of spiceypy, cite the specific zenodo doi.

Reply:
> should the message be in quotes?

No, YAML only needs quotes when the string contains disallowed characters, like `:`. I am actually removing a bunch of unnecessary quotes in the updates I will push shortly.

Reply:
Updated the message. Let me know if it's what you were going for.